client/src/_playground/styles.playground.module.css

```diff
@@ -0,0 +1,18 @@
+.componentsPlayground {
+  background-color: #f1f1f1;
```

Comment:
Background color could be a part of CSS theme perhaps?

Reply:
Yes it can be. I just set random, neutral color for a background for the tested components. TBH I treated this playground like PoC to show you this and take feedback from you all. But I will change it to already existing color variable :D

Reply:
I would be good to see all of those components on the background color we use everywhere else (in apps).